### PR TITLE
:seedling: add min pwd age to pwd reset flow description

### PR DESCRIPTION
**Pull request by mpeng-okta**
Adds description in password reset flow for minimum password age requirement
Updates repo to use okta/moment (necessary for TimeUtil)

#### 6 Hours
image
#### 3 Days
image

#### 6 Hours
image
#### 3 Days
image

## Primary Reviewer(s)
- @hor-kanchan-okta 
- @mauriciocastillosilva-okta 

Resolves: OKTA-131031

### CODE REVIEW

**Comment by mpeng-okta** [comment]
Sure, will do. I'll post screenshots once the i18n changes are in, so I can pull those in and show the proper messages.

**Comment by rchild-okta** [comment]
Btw @mpeng-okta if you closed/reopened to re-run Travis, there's another option - if you sign in to Travis with your github account, you'll get the ability to re-run (+ get notifications when tests are done, etc).

**Comment by mpeng-okta** [comment]
Wow, good read :D yeah, I'll go ahead and do that. Thanks!

File: test/unit/spec/PasswordReset_spec.js [hunk 1]
```diff
@@ -152,16 +170,40 @@ function (Q, _, $, OktaAuth, LoginUtil, Util, PasswordResetForm, Beacon, Expect,
...
+      });
+    });
+
+    itp('has a valid subtitle in minutes if only password age "minAgeInMinutes" defined', function () {
+      return setup({policyAge: 'minAgeInMinutes'}).then(function (test) {
+        expect(test.form.subtitleText()).toEqual(
+          'At least 30 minute(s) must have elapsed since you last changed your password.');
+      });
+    });
+
```
Comment (mauriciocastillosilva-okta) [inline test/unit/spec/PasswordReset_spec.js]: we need to add 2 more tests
- minAge combined with historyCount
- all options (```{policyComplexity: 'all', policyAge: 'all'}``` (or something similar)

File: webpack.common.config.js [hunk 2]
```diff
@@ -21,7 +21,7 @@ module.exports = function (outputFilename) {
         'okta/jquery': SHARED_JS + '/util/jquery-wrapper',
         'okta/underscore': SHARED_JS + '/util/underscore-wrapper',
         'okta/handlebars': 'handlebars/dist/handlebars',
-        'okta/moment': EMPTY,
+        'okta/moment': 'moment/moment',
```
Comment (mauriciocastillosilva-okta) [inline webpack.common.config.js]: @rchild-okta we need to add moment now, this probably will increase the size of the widget. Just FYI, unless you know a better way to add the time functionality that is needed)

**Pull request merged**